src/dataMigration.js

```diff
@@ -188,6 +188,46 @@ const dataMigrations = [
       });
     },
   },
+
+  //  2.3.6 Introduced a patch for ItemBatch outgoing sync supplier. Previously,
+  // all outgoing ItemBatches had their supplier field set to the default
+  // supplying store record for its store. 2.3.6 ensured that the supplier
+  // of a ItemBatch was correctly synced out, maintaining it's supplier field.
+  // This migration code
+  {
+    version: '2.3.6',
+    migrate: database => {
+      const supplierInvoiceType = 'supplier_invoice';
+      const finalisedStatus = 'finalised';
+      const itemBatches = database.objects('ItemBatch');
```

Comment:
Hmm, maybe this should only run for ItemBatches with quantity > 0

Reply:
makes sense, but before I do it can I just make sure, because I don't really know them well enough: in `outgoingSyncUtils`, `ItemBatch` records aren't filtered by quantity. Do you think having an `ItemBatch` with potentially the wrong supplier, even though `0` quantity will effect reporting at all?

Reply:
The reason i mention this was to minimise sync out records.

Maybe it's ok to do it for all batches but ONLY if otherParty has changed